packages/now-cli/src/util/deploy/process-deployment.ts

```diff
@@ -0,0 +1,121 @@
+import bytes from 'bytes';
+// @ts-ignore
```

Comment:
Something is seriously wrong if you are disabling typescript here. Does this fail on your machine or just Circle CI?

Reply:
Just Circle CI and only under coverage test. Locally and in integration tests it works fine